modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/display/context/ContentDashboardAdminDisplayContext.java

```diff
@@ -204,6 +213,12 @@ public boolean isAuditGraphEnabled() {
 		return _ffContentDashboardConfiguration.auditGraphEnabled();
 	}
 
+	private Map<String, Object> _getContext() {
+		return Collections.singletonMap(
+			"rtl",
```

Comment:
We usually use in portal 'textDirection' as the API field to say if the text is "rtl" or "ltr". We should return ltr is the textDirection is no RTL

Reply:
Yes! I saw it!, It makes sense more for me. @dgarciasarai, any problem if I change the boolean property "rtl" to a string property with the actual direction?

Reply:
For me, it's clearer to inject a boolean. Fewer checks on the frontend side.

Reply:
See: https://github.com/liferay/liferay-portal/blob/37fea2776526174f33fff458362221cdf0a6f492/modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/components/AuditBarChart.js#L161

Reply:
I see, but, for consistency, I think It's better to use "languageDirection" there are several examples in Portal

Reply:
Finally, I'm using "languageDirection" as it's been already used in several parts